### PR TITLE
Create nav bar

### PR DESCRIPTION
**Pull request by f117qaz123**

### CODE REVIEW

**Comment by f117qaz123** [comment]
[Create NavBar](https://trello.com/c/mJlpyoAI/7-create-navbar)

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
 const App = () => (
-    <>
-      <PlayersListTable />
-      <ClubList />
-    </>
-  );
-
+  <>
+    <NavBar/>
```
Comment (LianyuMa) [inline client/src/App.tsx]: any specific reason `<NavBar />` not inside `<BrowserRouter />`?
https://v5.reactrouter.com/web/api/BrowserRouter

Comment (f117qaz123) [reply client/src/App.tsx]: I will do the refactor

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-    <>
-      <PlayersListTable />
-      <ClubList />
-    </>
-  );
-
+  <>
+    <NavBar />
+    <Routes>
+      <Route path={url + '/'} element={<PlayersListTable />} />
```
Comment (LianyuMa) [inline client/src/App.tsx]:
```suggestion
      <Route path="/" element={<PlayersListTable />} />
```
this should be good enough, no?

Comment (f117qaz123) [reply client/src/App.tsx]: It works

**Review by LianyuMa — APPROVED** [review]

**Pull request merged**